src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -224,4 +246,75 @@ public function trigger_update_actions( $attendee_data ) {
 		 */
 		do_action( 'event_tickets_rsvp_after_attendee_update', $attendee_id, $post_id, $attendee_status );
 	}
+
+	/**
+	 * Bulk update multiple attendees with the same field values.
+	 *
+	 * More efficient than calling update() in a loop.
+	 *
+	 * Note: This method does NOT use database transactions. Updates are
+	 * performed independently and may result in partial success if some
+	 * updates fail.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $attendee_ids Attendee IDs.
+	 * @param array $updates      Fields to update (e.g., ['attendee_status' => 'yes']).
+	 *
+	 * @return array Results indexed by attendee ID (true = success, false = failure).
+	 */
+	public function bulk_update( array $attendee_ids, array $updates ): array {
+		$results = [];
+
+		foreach ( $attendee_ids as $attendee_id ) {
+			$result = $this->by( 'id', $attendee_id )
+							->set_args( $updates )
+							->save();
+
+			$results[ $attendee_id ] = false !== $result;
+		}
+
+		return $results;
+	}
+
+	/**
+	 * Get attendee counts grouped by RSVP status.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $event_id Event ID.
+	 *
+	 * @return array Status counts (e.g., ['yes' => 10, 'no' => 5]).
+	 */
+	public function get_status_counts( int $event_id ): array {
+		global $wpdb;
+
+		$results = $wpdb->get_results(
```

Comment:
I would prefer using our `StellarWP\DB\DB` instead of the `global $wpdb`. Reason is to avoid phpcs issue mostly.